client/src/main/java/com/tigrisdata/db/client/search/FacetCount.java

```diff
@@ -0,0 +1,28 @@
+package com.tigrisdata.db.client.search;
+
+import com.tigrisdata.db.api.v1.grpc.Api;
+import java.util.Objects;
+
+public class FacetCount {
+
+  private String value;
+  private long count;
```

Comment:
make this class immutable. as the state is not changing after it is constructed.